sdk/src/main/java/com/hedera/hashgraph/sdk/Transaction.java

```diff
@@ -729,6 +735,13 @@ protected void requireOneNodeAccountId() {
         }
     }
 
+    protected TransactionBody.Builder spawnBodyBuilder() {
```

Comment:
What about `toProtobufTransactionBodyBuilder()` This would be somewhat consistent with the rest of the SDK since we use `toProtobuf()` quite a bit. We just don't use the word `spawn` anywhere in the SDK.